### DISCUSSION TITLE
Error loading graph: cluster [XXX] is not found or is not accessible for Kiali

### DISCUSSION THREAD

**Discussion by pachas7** [kiali/kiali#7417] [Q&A]
### Describe the bug
Hi, we are trying to visualize a traffic graph on kiali in a environment where we have two clusters with istio installed in a mesh. 
After enable our application to send traffic from clusterA to clusterB, the traffic graph show the belows message "Cannot load the graph: cluster [XXXXX] is not found or is not accessible for Kiali".

### Expected Behavior
See the traphic graph with multicluster or with link to show the other kiali.

### Environment
- **Kiali version:1.82**
- **Istio version:1.18**
- **Kubernetes version:1.27** 
- **Other notable environmental factors:**

**Comment by jmazzitelli** [comment]
(_moving to Q&A discussion as it sounds like this is likely not a bug or was already fixed_)

**Comment by jmazzitelli** [comment]
You might want to see if this was already addressed - see this: https://github.com/kiali/kiali/issues/7305

**Reply by jshaughn** [reply]
+1, please try v1.85.
